src/lib/V3DutchOrderLib.sol

```diff
@@ -128,11 +156,13 @@ library V3DutchOrderLib {
     function hash(V3DutchOutput memory output) internal pure returns (bytes32) {
         return keccak256(
             abi.encode(
-                NON_LINEAR_DUTCH_OUTPUT_TYPE_HASH,
+                V3_DUTCH_OUTPUT_TYPE_HASH,
                 output.token,
                 output.startAmount,
                 hash(output.curve),
-                output.recipient
+                output.recipient,
+                output.minAmount,
+                output.adjustmentPerGweiBasefee
```

Comment:
is `gwei` enough granularity? think even partial gwei changes in basefee may be impactful

Reply:
Yeah, it's denominated in gwei but since the division by 1 gwei happens after the multiplication, we can take advantage of wei-level changes in the gas fee.
`int256 inputDelta = int256(order.baseInput.adjustmentPerGweiBaseFee) * gasDeltaGwei / 1 gwei`

See this test for example:
https://github.com/Uniswap/UniswapX/blob/c0d699883b02668aed1d95077efb4a78ec7d4cfc/test/reactors/V3DutchOrderReactor.t.sol#L1127